ArcGISRuntimeSDKQt_QMLSamples/Maps/ApplyScheduledMapUpdates/ApplyScheduledMapUpdates.qml

```diff
@@ -0,0 +1,210 @@
+                console.log("Check for updates complete");
+
+                // if updates are available, enable the download button & set detail text
+                if (checkForUpdatesResult.downloadAvailability === Enums.OfflineUpdateAvailabilityAvailable) {
+                    // enable the button
+                    downloadUpdatesButton.enabled = true;
+
+                    // set the text
+                    availabilityText.text = "Updates Available";
+                    availabilityDetailsText.text = "Updates size: %1 bytes".arg(offlineMapSyncTask.checkForUpdatesResult.scheduledUpdatesDownloadSize);
+                } else {
+                    // disable the button
+                    downloadUpdatesButton.enabled = false;
+
+                    // set the text
+                    availabilityText.text = "No updates available";
+                    availabilityDetailsText.text = "The preplanned map area is up to date"
+                }
+
+                busyIndicator.visible = false;
+            }
+        }
+
+        onCreateDefaultOfflineMapSyncParametersStatusChanged: {
+            if (createDefaultOfflineMapSyncParametersStatus === Enums.TaskStatusInProgress) {
+                console.log("Create Parameters in progress...");
+            } else if (createDefaultOfflineMapSyncParametersStatus === Enums.TaskStatusErrored) {
+                console.log("Create Parameters error:", error.message, error.additionalMessage);
+            } else if (createDefaultOfflineMapSyncParametersStatus === Enums.TaskStatusCompleted) {
+                console.log("Create Parameters complete");
+
+                // get the parameters result
+                const params = createDefaultOfflineMapSyncParametersResult;
+
+                // set the parameters to download all updates for the mobile map packages
+                params.preplannedScheduledUpdatesOption = Enums.PreplannedScheduledUpdatesOptionDownloadAllUpdates;
+
+                // set the map package to rollback to the old state should the sync job fail
+                params.rollbackOnFailure = true;
+
+                // create the job
+                syncJob = offlineMapSyncTask.syncOfflineMap(createDefaultOfflineMapSyncParametersResult);
+
+                // connect to the job signals
+                syncJob.jobStatusChanged.connect(()=>{
+                                                     if (syncJob.jobStatus === Enums.JobStatusSucceeded) {
+                                                         const mapResult = syncJob.result;
+
+                                                         if (mapResult.mobileMapPackageReopenRequired) {
+                                                             // close and reopen the MMPK if required
+                                                             mmpk.close();
+                                                             mmpk.load();
```

Comment:
as  discussed, this needs a whole new mmpk instance - you can't just `load` the old one again